docker-compose.yaml

```diff
@@ -7,13 +7,14 @@ services:
     ports:
       - "8085:8085"
     environment:
-      - CDK_MINTD_URL=https://example.com
+      - CDK_MINTD_URL=http://192.168.1.188:8085
       - CDK_MINTD_LN_BACKEND=fakewallet
       - CDK_MINTD_LISTEN_HOST=0.0.0.0
       - CDK_MINTD_LISTEN_PORT=8085
       - CDK_MINTD_MNEMONIC=
       - CDK_MINTD_DATABASE=redb
       - CDK_MINTD_CACHE_BACKEND=memory
+      - CDK_MINTD_MNEMONIC=essence thing digital again unaware famous case usage friend claw picnic donor
```

Comment:
Imagine you just forgot to remove this in testing. I don't want a seed phrase in the as we've seen in other projects some people will just run it with that.